R/data_unite.R

```diff
@@ -0,0 +1,111 @@
+#' @title Unite ("merge") multiple variables
+#' @name data_unite
+#'
+#' @description
+#' Merges values of multiple variables per observation into one new variable.
+#'
+#' @param data A data frame.
+#' @param new_column The name of the new column, as a string.
+#' @param separator Separator, as string, to use between values.
+#' @param append Logical, if `FALSE` (default), removes original columns that
+#' were united. If `TRUE`, all columns are preserved and the new column is
+#' appended to the data frame.
```

Comment:
Actually I think it would make more sense to go the other way and to have an argument `remove` that is `TRUE` by default (just to stick to `tidyr::unite()`)

Reply:
I thought about this, too, but the logic in our package is to `append` (or not to). We can _additionally_ have a `remove` argument that simply defaults to `!append`?

Reply:
It's better to only have one of the two so we can keep `append` for consistency with the rest